sdk/opendp/smartnoise/evaluation/_aggregation.py

```diff
@@ -79,27 +78,23 @@ def dp_var(self, df, colname):
 
     def dp_mechanism_count(self, df, colname):
         """
-        Returns repeatedly applied noise adding mechanisms
-        like Laplace and Gaussian available in SmartNoise-SDK to count query
+        Returns repeatedly applied noise adding mechanism to count query
         """
         exact_count = df[colname].count()
-        mech = Laplace(self.epsilon)
         if(self.mechanism == "Gaussian"):
             mech = Gaussian(self.epsilon)
         return np.array([mech.release([exact_count]).values[0] for i in range(self.repeat_count)])
 
     def dp_mechanism_sum(self, df, colname):
         """
-        Returns repeatedly applied noise adding mechanisms
-        like Laplace and Gaussian available in SmartNoise-SDK to sum query.
+        Returns repeatedly applied noise adding mechanisms to sum query.
         Sensitivity is set as absolute difference between max and min values
         within the column
         """
         exact_sum = df[colname].sum()
         M = float(abs(max(df[colname]) - min(df[colname])))
-        mech = Laplace(self.epsilon, sensitivity = M)
         if(self.mechanism == "Gaussian"):
-            mech = Gaussian(self.epsilon)
+            mech = Gaussian(self.epsilon, 10E-5, M)
```

Comment:
We should remove the configurability of sel.mechanism. Otherwise passing Laplace would end up in an undefined error for mech which would be hard to debug

Reply:
Good call; I nuked it all